docs/Using-Pantheon/Accessing-Logs-Using-JSON-RPC.md

```diff
@@ -0,0 +1,164 @@
+# Accessing Logs Using JSON-RPC API Methods
+
+Access logs using JSON-RPC API methods:
+ 
+* [`eth_getFilterChanges`](../Reference/JSON-RPC-API-Methods.md#eth_getfilterchanges)
+* [`eth_getFilterLogs`](../Reference/JSON-RPC-API-Methods.md#eth_getfilterlogs)
+* [`eth_getLogs`](../Reference/JSON-RPC-API-Methods.md#eth_getlogs)
+
+Before using [`eth_getFilterChanges`](../Reference/JSON-RPC-API-Methods.md#eth_getfilterchanges) and 
+[`eth_getFilterLogs`]((../Reference/JSON-RPC-API-Methods.md#eth_getfilterlogs)), create the filter 
+using [`eth_newFilter`](../Reference/JSON-RPC-API-Methods.md#eth_newfilter).
+
+!!! tip
```

Comment:
well, tips are things you could do to improve or speed up your usage like keyboard shortcuts or "tip: use a config file instead of re-writing all the command line options each time" or "tip: you can use a hex to dec online converter to read value easily" something like that IMO.

Reply:
Fixed and agree with Nico tips do have their place but agree with both of you, this isn't one.